modules/local/prelude/main.nf

```diff
@@ -0,0 +1,335 @@
+import groovy.xml.XmlSlurper
+
+process SUMMARY_XML {
+    tag "$meta.id"
+    label 'process_single'
+
+    input:
+    val(meta)
```

Comment:
The standard in nf-core modules to take a meta map and value is `tuple val(meta), val(foo)`. Maybe it's not required to keep local modules in line with that, but it would be nice to be consistent if possible. Same goes for the other processes here.